CHANGELOG.md

```diff
@@ -70,6 +70,10 @@ It should be noted that some of these features break backwards compatibility. Se
 
 - Parallelism is improved when running the same target with different arguments in certain cases (e.g. the target uses `WITH DOCKER`).
 - Fixed a log sharing upload-resumption bug
+- Fixed multiple issues with the lexer failing to parse certain characters in shell command substitution (`$()`) and single quoted strings.
```

Comment:
this should go under `unreleased` since it wasn't fixed in v0.8.0-rc2

Reply:
Updated.

Reply:
Oh, interesting. It turns out this hasn't been working in `main`. I noticed this while testing and thought it must have been due to the lexer changes. I'll remove it. 

https://github.com/earthly/earthly/actions/runs/7561776859/job/20590798774#step:4:35